tn_api_pricing_booking_spec.yml

```diff
@@ -878,35 +953,33 @@ components:
                             description: Email address of travel agent who created the booking
                             type: string
                             example: agent@tripninja.io
-                        booking_date: 
-                            description: Date on which the booking was made
-                            type: string
-                            example: "2019-07-07"
-                        departure_date: 
-                            description: Date on which the first flight of the trip leaves.
-                            type: string
-                            example: "2019-11-21"
-                        flights: 
-                            description: List of all flights segment IATA codes
-                            type: array
-                            example: ['YHZ-YOW', 'YOW-YYZ', 'YYZ-YHZ']
-                        customer_email: 
-                            description: Email address of customer
-                            type: string
-                            example: customer@email.com
-                        customer_name: 
-                            description: Name of the primary passenger
-                            type: string
-                            example: customer@email.com
-                        total_price: 
-                            description: Total price of the trip including all taxes, fees, and markup.
-                            type: float
-                            example: 399.96
-                        currency: 
-                            description: Currency in which the booking was made
-                            type: string
-                            example: CAD
-                        payment:       
-                            type: boolean
-                            description: Notes whether the booking included payment information. 
-                            example: true
+
+    passenger:
```

Comment:
Based on the design we want a few more passenger details like email, phone, passport # - I dont remember which ones we decided on though.....
![image](https://user-images.githubusercontent.com/27984048/88167102-af2f7780-cbee-11ea-95f2-58064e19c431.png)

Reply:
Done